contrib/Docker_Grafana_Blueflood/docker-entrypoint.sh

```diff
@@ -37,6 +37,8 @@ blueflood:
   apikey: $RAX_APIKEY                     
   authentication_module: rax_auth
   authentication_class: BluefloodAuth
+  enable_submetrics: true
+  submetric_aliases: { "_sum": 'sum' }
```

Comment:
@ChandraAddala I see.  I didn't realize that it is not regular behavior of Grafana.  For this PR, the goal is to provide a replacement for the Enum support that we are going to take out.  So does this mean that we need to document the submetrics (and make releases if needed) before we can promote a better way for uptime reporting?